{
    "type": "bundle",
    "id": "bundle--f9775604-a86c-42ca-8198-6b8cde80c04d",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--1385ccfa-ccd5-4db1-85ae-11fa0375751c",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-04-03T00:20:54.279843Z",
            "modified": "2025-04-03T00:20:54.279843Z",
            "name": "CVE-2025-24240",
            "description": "A race condition was addressed with additional validation. This issue is fixed in macOS Ventura 13.7.5, macOS Sequoia 15.4, macOS Sonoma 14.7.5. An app may be able to access user-sensitive data.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-24240"
                }
            ]
        }
    ]
}